types/Environment.js

```diff
@@ -12,8 +12,12 @@ import type {Global} from './Global';
 import type {Script} from 'vm';
 import type {ModuleMocker} from 'jest-mock';
 
+export type EnvironmentOptions = {
+  console?: Object,
+};
+
 declare class $JestEnvironment {
-  constructor(config: ProjectConfig): void;
+  constructor(config: ProjectConfig, options?: EnvironmentOptions): void;
```

Comment:
I expect in the next major version we can make this a required argument, but thought it best to leave it as an optional argument for now. I default to an empty object in the jsdom environment.